public/js/init.js

```diff
@@ -514,6 +516,7 @@ async function init() {
   // TODO(kilemensi): Load other data here
 
   handleLocationChange('nairobi');
+  window.analytics.getAnalytics('Nairobi');
```

Comment:
Lets choose one (`slug` or `Name`) and stick to it for all of the `handleLocationChange` methods.

Reply:
Done